extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -180,32 +171,23 @@ export class DoubleclickA4aEligibility {
     }
     let experimentId;
     const urlExperimentId = extractUrlExperimentId(win, element);
-    let experimentName = DFP_CANONICAL_FF_EXPERIMENT_NAME;
     if (!this.isCdnProxy(win)) {
       // Ensure that forcing FF via url is applied if test/localDev.
       if (urlExperimentId == -1 &&
           (getMode(win).localDev || getMode(win).test)) {
         experimentId = MANUAL_EXPERIMENT_ID;
       } else {
         let unconditionedExp;
-        // For unconditioned canonical experiment, in the experiment branch
-        // we allow Fast Fetch on non-CDN pages, but in the control we do not.
+        // For unconditioned canonical holdback, in the control branch
+        // we allow Fast Fetch on non-CDN pages, but in the experiment we do not.
         if ((unconditionedExp = getExperimentBranch(
```

Comment:
Can collapse this where we allow FF unless its the holdback exp:

return getExperimentBranch(...) != DOUBLECLICK_UNCONDITIONED_EXPERIMENTS.CANONICAL_HLDBK_EXP;

Reply:
done